examples/CMakeLists.txt

```diff
@@ -22,6 +42,9 @@ function(add_example file mode lib)
     target_link_libraries(${target} PUBLIC ${lib})
 	set_property(TARGET ${target} PROPERTY LANGUAGE CXX)
 	set_property(TARGET ${target} PROPERTY CXX_STANDARD 17)
+	if("${target}" STREQUAL "z_bytes_zenohc")
```

Comment:
this should also work for pico backend, right ?

Reply:
yes, right, added both backend support